.github/workflows/build-and-publish-rule-book.yaml

```diff
@@ -140,3 +175,43 @@ jobs:
           publish_branch: ${{ matrix.language }}
           publish_dir: ./${{ matrix.language }}
           force_orphan: true
+
+  check_pages:
+    runs-on: ubuntu-latest
+    needs: build
+    strategy:
+      fail-fast: false
+      matrix:
+        language: ["pl", "es", "fr", "ru"]
```

Comment:
We have quite a few places already with this list... Will need to do some deduplication of these lists as follow-up

Reply:
Yeah, good idea.